config/peerpods/podvm/bootc/nvidia/generate-nvidia-cdi.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#load drivers
+nvidia-ctk -d system create-device-nodes --control-devices --load-kernel-modules
+
+#nvidia-persistenced
+# set confidential compute to ready state
+#nvidia-smi conf-compute -srs 1 
+# Generate NVIDIA CDI configuration
```

Comment:
the commented are required for CoCo, i'm not sure how to differentiate

Reply:
This is a service running inside the VM I'm not sure how to do it so it will be taken into account at runtime